src/agent/Cargo.toml

```diff
@@ -25,7 +25,7 @@ tokio = { version = "1.2.0", features = ["rt", "rt-multi-thread", "sync", "macro
 futures = "0.3.12"
 netlink-sys = { version = "0.6.0", features = ["tokio_socket",]}
 tokio-vsock = "0.3.1"
-rtnetlink = "0.7.0"
+rtnetlink = { git = "https://github.com/little-dude/netlink", rev = "a9367bc4700496ddebc088110c28f40962923326" }
```

Comment:
It's better to add an detail comments here to note why we use the github repo instead of the create.io crate, and once the new version released, we would switch to it.

Reply:
done